src/UICrafter.Mobile/Components/Layout/MainLayout.razor

```diff
@@ -11,35 +11,13 @@
 <MudSnackbarProvider />
 
 <div class="page">
- @*    <div class="sidebar">
-        <NavMenu />
-    </div> *@
-
     <main class="main">
-        <div class="top-row px-4">
-            <AuthorizeView>
-                <Authorized>
-                    Hello, @context.User.Identity?.Name
-                    <a href="#" @onclick="Logout">Log out</a>
-                </Authorized>
-                <NotAuthorized>
-                    <a href="#" @onclick="Login">Log in</a>
-                </NotAuthorized>
-            </AuthorizeView>
-        </div>
-
-        <article class="content px-4">
-            @Body
-        </article>
+        @Body
     </main>
-
-    <AuthorizeView>
-        <Authorized>
-            <footer class="footer">
-                <NativeNavbar />
-            </footer>
-        </Authorized>
-    </AuthorizeView>
+    <footer>
+        <NativeNavbar Class="native-navbar-fixed" />
+        <NativeNavbar Style="visibility: hidden;" />
```

Comment:
Why 2 of them?

Reply:
This is to offset the body with the size of the NativeNavBar Alternative to this is that I would need to add margin to the body or something but I dont know about the exact size so I think a hidden NavBar is a better choice.